components/multi-attribute-login/org.wso2.carbon.identity.unique.claim.mgt/src/main/java/org/wso2/carbon/identity/unique/claim/mgt/listener/UniqueClaimUserOperationEventListener.java

```diff
@@ -99,6 +99,22 @@ public boolean doPreAddUser(String userName, Object credential, String[] roleLis
         return true;
     }
 
+    @Override
+    public boolean doPostAddUser(String userName, Object credential, String[] roleList, Map<String, String> claims,
+                                 String profile, UserStoreManager userStoreManager) throws UserStoreException {
+
+        if (!isEnable()) {
+            return true;
+        }
+
+        try {
+            checkClaimUniqueness(userName, claims, profile, userStoreManager, credential);
```

Comment:
```suggestion
            checkClaimUniqueness(userName, claims, profile, userStoreManager, null);
```

Reply:
Since we have already validated the password in preAdd handler, no need to do it again.